### PR TITLE
Use releases for releases

### PR DESCRIPTION
**Pull request by sethvargo**

### CODE REVIEW

**Comment by sethvargo** [comment]
Looks like tests failed on tip, but they are marked as allowed failures?

**Comment by mitchellh** [comment]
Weird that allowed failures make it an X o.O

File: website/Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
 
-ruby "2.2.3"
+ruby "2.2.2"
```
Comment (mitchellh) [inline website/Gemfile]: I think Otto has to be 2.2.3 for some reason, but I'll play with that.

**Comment by mitchellh** [comment]
LGTM

**Pull request merged**